server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+const { Availability } = require("./Availability");
+
+const profileSchema = new mongoose.Schema({
+  first_name: {
```

Comment:
```suggestion
  firstName: {
```

Ditto throughout the schema 🚀